docs/introduction.tex

```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
 Problem definition
 
 =========================GWEN=========================
-\begin{itemize}
-    \item definition of NFC
-    \item Quickness vs accuracy
-    \item Depletion calculations for LWRs
-    \item current practice for depletion calculations [citations]
-    \item pros and cons of those practices [citations]
-    \item Does it even matter? Fuel cycle simulation is a system-level analysis. How much does it matter? [citations]
-\end{itemize}
-
-
-\gls{NFC} simulations are system-level analyses that track
-material flow in a \gls{NFC}. 
-
-Since \gls{NFC} simulations
-involve many facilities, the fidelity and detail of modeling each
-facility are sacrificed for quickness. 
-
-One of the most crucial functionalities in a \gls{NFC}
-simulation is the transmutation of nuclear fuel in a reactor,
-which is directly related to the \gls{UNF} composition
-and waste / material profile.
-
-Depletion calculations for fuel cycle simulations in current
-\gls{NFC} simulators are either
-recipe based (no calculation performed), 
-library based, [so on and so forth, with citation].
 
+The \gls{NFC} is a complex system of facilities and material 
+mass flows that are combined to provide nuclear energy for use 
+in society, usually in the form of electricity 
+\cite{yacout_modeling_2005}. 
+\gls{NFC} simulators are system analysis tools used to investigate 
+issues related to the dynamics of a nuclear fuel cycle in both 
+high and low resolution. 
+An example of a high resolution element is the spent fuel 
+isotopic composition from a single fuel bundle, and an example 
+of a low resolution element is the total fuel utilization in 
+the system. 
+The intention behind the use of \gls{NFC} simulators is to develop 
+a better understanding of the dependence between various components 
+in the system and the effects of changes on the system. 
+Their goal is to assist in evaluating and improving potential 
+strategies for nuclear power development in terms of improving waste 
+management, economic competitiveness, etc \cite{yacout_modeling_2005}.   
+
+One of the major functionalities of a \gls{NFC} simulator is its 
+ability to transmute nuclear fuel in a reactor based on reactor 
+conditions such as burnup, enrichment, etc. 
+The transmutation results impact the accuracy of the \gls{UNF} 
+composition, and thus the capability of using the \gls{NFC} to 
+analyze the impact of a \gls{NFC} on variables such as waste profile.  
+
+Current fuel cycle simulator codes include \Cyclus, ORION, DYMOND 
+and VISION. 
+Each of these NFC simulators obtain transmutation results using 
+different methods. 
+Some of these codes have multiple methods for obtaining 
+transmutation results. 
+There is essentially two overall types of methods to obtain transmutation 
+results: recipe-based method and library-based method. 
+The recipe-based method is a low fidelity method, while the library-based method is 
+a high fidelity method. 
```

Comment:
I think this sort of binary division into low and high fidelity is not the best option. Maybe just get rid of the sentence?